tools/bin/hawq_ctl

```diff
@@ -149,7 +153,7 @@ class HawqInit:
               (source_hawq_env, self.vseg_num_per_node)
         result = local_ssh(cmd, logger)
         if result != 0:
-            logger.warn("Set hawq_resourcemanager_query_vsegment_number_per_segment_limit failed.")
+            logger.warn("Set hawq_resourcemanager_query_vsegment_number_per_segment_limit failed")
         return result
```

Comment:
hawq_resourcemanager_query_vsegment_number_per_segment_limit guc name change, you should use new guc name